README.md

```diff
@@ -1,98 +1,213 @@
 # webdriver-recorder
 This enhances selenium's webdriver to simplify the interface and capture
 screenshots along the way. When run it will output html with screenshots
-of different scenarios. This uses python3.4+ only.
+of different scenarios.
+
+## Add this to your project
+
```

Comment:
Maybe a link here to the First-time developer setup section? Something like, if you don't have this installed or if its your first time, refer to First-time developer setup for details?

Reply:
These two things are not the same.

"Add this to your project" is for people who just want to use the plugin, who don't need to install anything (except Chromedriver). It is pretty common for dependencies like chromedriver to just tell people it is a prerequisite and let them handle setting up their environment. These people may be anyone in the world, not just IAM folks, so we do _not_ want to instruct them to install poetry, run our scripts, etc, because they will not have the repository, just the installed dependency. 

The first-time developer setup section should only be followed by people who need to make changes to the package itself.

Reply:
Even Selenium only goes so far as to say "Make sure you have chromedriver installed" in their docs (https://chromedriver.chromium.org/getting-started) -- we should take no greater responsibility than that :)